common-theme/layouts/partials/issues.html

```diff
@@ -1,69 +1,76 @@
 {{/* This is a page, basically, not a block, so we don't go via blockData for now */}}
 {{ $repo := .Params.backlog }}
-{{ $issuesUrlBase := print $.Site.Params.orgapi $repo "/issues?per_page=100&state=open&direction=asc&page=" }}
+{{ $issuesUrl := print $.Site.Params.orgapi $repo "/issues?per_page=100&state=open&direction=asc" }}
 {{ $filter := .Params.backlog_filter }}
 {{ $currentPath := .Page.RelPermalink }}
 {{ $headers := partial "github-auth.html" }}
-{{/* <!-- walk through paginated issues --> */}}
-{{ .Page.Scratch.Set "issues" slice }}
-{{ partial "fetch-paginated-issues.html" (dict "Scratch" .Page.Scratch "issuesUrlBase" $issuesUrlBase "currentPath" $currentPath "page" 1 "headers" $headers) }}
-{{ $issues := .Page.Scratch.Get "issues" }}
 
-{{/* <!-- if no object show error --> */}}
-{{ if ne $issues nil }}
-  <!-- range over issues list and pull out useful data -->
-  {{ range $issues }}
+{{ with resources.GetRemote $issuesUrl $headers }}
+  {{ with .Err }}
+    {{ errorf "Failed %s on %s. Error: %s" $issuesUrl .Page.Title . }}
+  {{ end }}
+
+  {{ with .Content }}
+    {{ if eq . "[]" }}
+      {{ warnf "No issues found at %s" $issuesUrl }}
```

Comment:
Maybe error rather than warn? And I'd be tempted to have this as an accumulated check after the loop (i.e. apply the filter in the loop, if we show an issue set something to true, and error if we didn't show at least one issue) - "You set a filter and it ignored all issues" feels like as much of an error as "Your original query without filters had not results"?

Reply:
So, the question here is how much do we want builds to fail, basically, based on external factors.

If there are no issues set on a repo we aren't personally in charge of, like a course we are not running, should that fail our build? When PD can't merge their blocks because there's a no response error they cannot and do not fix it. Instead I have to run around and fix it and a lot of time is wasted on this sort of thing that I don't necessarily have. 

I don't disagree that it would be more rigorous, it's just that there's only me here.